docs/content/content-blocks.md

```diff
@@ -173,13 +286,12 @@ Here's my figure:
 {glue:}`sorted_means_fig`
 ```
 
-See {doc}`glue` for more information on how to use Glue to insert your outputs
-directly into your content.
+See [](content:code-outputs:glue) for more information on how to use Glue to insert your outputs directly into your content.
```

Comment:
The `[]` takes the title from the target document right?

Reply:
yep 👍

Reply:
Should maybe document that in citations.md

Reply:
That sounds good -- I didn't know you could do that.

Reply:
added 👍